components-api/src/main/scala/pl/touk/nussknacker/engine/api/component/ComponentProvider.scala

```diff
@@ -61,6 +61,11 @@ trait BoundedStreamComponent { self: Component =>
     AllowedProcessingModes.SetOf(ProcessingMode.BoundedStream)
 }
 
+trait StreamComponent { self: Component =>
```

Comment:
Is it obvious that a "stream" component is such that is allowed for both streaming and batch? :) https://martinfowler.com/bliki/UbiquitousLanguage.html As a representative of domain experts I want to file my objection ;-) I think that the only unambiguous would sth like `BothBoundedUnboundedCompent`. But when we add it, someone can say: why there is no `BothBoundedRequestResponseCompent` and `BothUnbounedRequrestResponseCompent`. What we would say than? :)

Reply:
Are Bounded and Unbounded now in our "Ubiquitous Language"? I don't think so ;) I agree with rest part of your opinion that someone can ask questions ;) I can agree and we can resign from this change..